tasks/49_cat-and-dog-years/cat-and-dog-years.js

```diff
@@ -0,0 +1,19 @@
+export function humanYearsCatYearsDogYears(humanYears) {
+  let catYears = 15;
+  let dogYears = 15;
+  let years = [humanYears, catYears, dogYears];
+
+  while (-- humanYears) {
```

Comment:
"while (-- humanYears)" -> looks like mistake and can raise questions.
Did writer wanted --humanYears or humanYears-- ?

Better use "humanYears -= 1" to avoid confusion